cumulusci/cli/cli.py

```diff
@@ -647,6 +655,10 @@ def task_run(config, task_name, org, o, debug):
     if exception:
         raise exception
 
+    if debug_after:
```

Comment:
Seems this should be before raising exception so user can inspect. Not sure if you had a reason in mind though.